clinics/manager/manager.go

```diff
@@ -160,45 +161,23 @@ func (c *manager) FinalizeMerge(ctx context.Context, sourceId, targetId string)
 
 	// Append share codes of source clinic
 	if source.ShareCodes != nil {
-		err = c.clinics.AppendShareCodes(ctx, targetId, *source.ShareCodes)
+		err = c.clinicsService.AppendShareCodes(ctx, targetId, *source.ShareCodes)
 		if err != nil {
 			return err
 		}
 	}
 
-	// Refresh patient count of target clinic
-	return c.refreshPatientCount(ctx, targetId)
+	_ = c.clinicsService.RefreshPatientCount(ctx, targetId) // Ignore any error, already logged
+
+	return nil
 }
 
 func (c *manager) GetClinicPatientCount(ctx context.Context, clinicId string) (*clinics.PatientCount, error) {
```

Comment:
The duplication of clinic service methods in the clinic manager seems unnecessary to me. It would only lead to confusion, IMO. You've added a couple of methods to the clinic manager that already exist in the clinic service. Is there some need for that duplication that I'm not seeing?

For background, in my head, the clinic manager existed as a way to sort of get around the patient/clinic circular dependency (I *think* Todd once confirmed this in a Slack convo). It was the place for functionality that required both the patient and the clinic repos. That's nothing formal, just what I had picked up from working on clinic. That's part of why duplicating methods from one to the other strikes me as odd.

I think you wrote a bit about this in Slack, I'll have to dig that thread up...